### PR TITLE
created a system admin role; added script to create/delete system admins

### LINKED ISSUE #2964: [Dev] Update Admin Role
The Admin role will be used by the development team to inspect issues that users might be seeing. Update the Admin role to have all of the view-only activities in the system. Create a database script script for adding a user as an Admin to all states and federal affiliations. Create a database script for removing an Admin user from all states and federal affiliations. Update tests a needed to manage new role.

### This task is done when…
 - [x] the Admin role has been updated to only have view-only activities
 - [x] there is a database script to add an Admin user to all states and federal affiliations*
 - [x] there is a database script to remove an Admin user from all state and federal affiliations*
 - [x] create a sysadmin user on test okta 
 - [x] add sysadmin users to seeds for new dev environments, test, and PRs
 - [x] create a sysadmin user on impl okta
 
*the scripts just need to be runnable on postgres, not a migration that is run on every environment. it will just be run whenever someone joins or leaves the team

### PR DESCRIPTION
**Pull request by thetif**
resolves #2964 

**Description-**
Creates an eAPD System Admin role to be used by team members to help in administrating the system. eAPD System Admin role will have approved affiliations with all of the states and have all of the view-* activities.

**This pull request changes...**

- creates an eAPD System Admin role with all of the view-* roles
- adds a sysadmin role to the set-up-users seed file so that it will be automatically added on dev, test, and PR environments
- creates a postgresql script to add a user as an eAPD System Admin role to all states
- creates a postgresql script to remove a user from all states

**This pull request was tested in the follow ways…**

- ran the seed script locally and it created the affiliations for the sysadmin user as expected

**Steps to manually verify this change...**

1. login to PR using new sysadmin user in keybase
2. should be taken to the Alaska APD list

### This pull request is ready to review when...

- [X] Changelog is updated as appropriate

### This pull request can be merged when…

- [ ] Code has been reviewed by someone other than the original author
- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: api/seeds/shared/set-up-users.js [hunk 1]
```diff
@@ -34,7 +40,14 @@ const createUsersToAdd = async (knex, oktaClient) => {
     .then(role => role.id);
 
   logger.info('Setting up affiliations to add');
-  const oktaAffiliations = [];
+  const oktaAffiliations = states.map(state => ({
```
Comment (tbolt) [inline api/seeds/shared/set-up-users.js:43]: This makes sense given our datamodel being "State-based" but it does seem strange to add 50 rows to a table to make someone a system admin.

**Review by tbolt — APPROVED** [review]
Works as advertised 😄

File: api/seeds/shared/set-up-users.js (hunk 1 above)
Comment (knollfear) [reply api/seeds/shared/set-up-users.js]: We have "federal" as a state.  I did some other work that just didn't append a state filter if the state was "federal".  IDK if that would work or make sense here.

**Review by NAretakis — APPROVED** [review]
Good to go.

**Pull request merged**